website/src/libs/api-v3/amazonPayments.js

```diff
@@ -70,24 +46,15 @@ api.authorize = (inputSet) => {
   });
 };
 
-api.closeOrderReference = (inputSet) => {
-  let amzPayment = connect(amazonPayments);
-  return new Promize((resolve, reject) => {
-    amzPayment.offAmazonPayments.closeOrderReference(inputSet, (err, response) => {
-      if (err) return reject(err);
-      return resolve(response);
-    });
-  });
-};
-
-api.executePayment = (inputSet) => {
```

Comment:
Great, I can add it back in. What is the form of that route, `POST /amazon/executePayment`? I do not find it in this router: https://github.com/HabitRPG/habitrpg/blob/develop/website/src/routes/payments.js